src/Tizen.NUI/src/public/Color.cs

```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
             }
         }
 
+        /// <summary>
+        /// The conversion constructor from an System.Drawing.Color instance.
+        /// </summary>
+        /// <param name="color">System.Drawing.Color instance</param>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
please remove <since_tizen> tag if this is hidden api.

Reply:
DONE